### DISCUSSION TITLE
Trying to blur with opacity instead of with transparency modifier on bg-color

### DISCUSSION THREAD

**Discussion by cdyrssen** [tailwindlabs/tailwindcss#13880] [Help]
I have dug into this issue and it seems to be a limitation of css in general. However, I'm hoping someone here can help me solve this issue. I have the below html code:

```
<!DOCTYPE html>
<html lang="en">

<head>
	<meta name="description" content="Example issue" />
	<meta charset="utf-8">
	<title>Example issue</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="author" content="christoffer-dyrssen">
	<script src="https://cdn.tailwindcss.com"></script>
</head>

<body style="background-image: url('https://cdn.pixabay.com/photo/2019/03/03/20/23/background-4032775_1280.png'" class="bg-cover">

	<div class="bg-neutral-400/80 w-1/2 m-auto mt-10 backdrop-blur">
 		<div class="hover:bg-neutral-500 opacity-80 hover:scale-y-150 hover:scale-x-105 p-1 text-center">
			Test text1
		</div>
 		<div class="hover:bg-neutral-500 opacity-80 hover:scale-y-150 hover:scale-x-105 p-1 text-center">
			Test text2
		</div>
 		<div class="hover:bg-neutral-500 opacity-80 hover:scale-y-150 hover:scale-x-105 p-1 text-center">
			Test text3
		</div>
	</div>

	<div class="bg-neutral-400 opacity-80 w-1/2 m-auto mt-10 backdrop-blur">
 		<div class="hover:bg-neutral-500 hover:scale-y-150 hover:scale-x-105 p-1 text-center opacity-100">
			Test text1
		</div>
 		<div class="hover:bg-neutral-500 hover:scale-y-150 hover:scale-x-105 p-1 text-center opacity-100">
			Test text2
		</div>
 		<div class="hover:bg-neutral-500 hover:scale-y-150 hover:scale-x-105 p-1 text-center opacity-100">
			Test text3
		</div>
	</div>
</body>
</html>
```
 I want to be able to have the hover states to act like when using opacity where the parent is not able to be seen through the children after scaling them up with opacity. However, I would also like to apply a blur to my background behind hovered and unhovered elements. It seems like since the transparency modified for bg-color applies an rgba(), the backdrop-blur filter can be applied. However, this also allows the parent to be seen through the child. If I use opacity, the child only shows the body's background image and not the parent's background, but then I am unable to do a backdrop blur because the rgba alpha channel is not used. Does anyone know how I can accomplish a combination of the affects? I provided screenshots below depicting the differences.

opacity and parent is not seen through child, but blur doesn't apply:
![opacity](https://github.com/tailwindlabs/tailwindcss/assets/22877684/89b555b5-a6a8-4fef-8022-93447b5d166d)

rgba and blur applied, but parent is seen through child:
![rgbablur](https://github.com/tailwindlabs/tailwindcss/assets/22877684/14730b83-1ed0-443c-a43c-03d493e26317)

**Comment by wongjn** [comment]
You could consider putting the backdrop blur on the children: https://play.tailwindcss.com/8zwHi6YJbR

**Reply by cdyrssen** [reply]
The issue I ran into applying the effects to the children is their edges become visible (see below image taken from you example). Also during transitions, with how the browser renders the elements, these edges are even more visible due to some aliasing or hysteresis (see [this example](https://play.tailwindcss.com/5DXE3Bdsmu) and adjust the preview size).

Backdrop blur applied to children:
![bluronchildren](https://github.com/tailwindlabs/tailwindcss/assets/22877684/b649beba-f962-4559-a17f-7221961d4424)

I am able to reduce other visual oddities like these when I apply the background effects to the parent instead.
